src/main/java/devcource/hihi/boardjpa/service/UserService.java

```diff
@@ -0,0 +1,47 @@
+package devcource.hihi.boardjpa.service;
+
+import devcource.hihi.boardjpa.domain.User;
+import devcource.hihi.boardjpa.dto.user.CreateUserDto;
+import devcource.hihi.boardjpa.dto.user.ResponseUserDto;
+import devcource.hihi.boardjpa.dto.user.UpdateUserDto;
+import devcource.hihi.boardjpa.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
p1;
스프링 빈을 주입할 때 필드 주입 대신 생성자 주입을 사용하시는 건 어떤가요??

빈 주입 방법에 대해서 찾아보시면 좋을 것 같아요!

Reply:
p1; 
```suggestion
    private final UserRepository userRepository;
```
생성자 주입을 사용한다면 final을 붙여주면 좋아요~